### ISSUE TITLE
Add Typescript definitions

### ISSUE THREAD

**Issue by egoist** [cacjs/cac#8]

**Comment by egoist** [comment]
I'm rewriting this in typescript 😄

**Comment by mitolog** [comment]
Hi, I'd like to use cac on my own typescript project. But I got an error when transpiling where `import` is written as below:

```js
import cac from 'cac';
const cli = cac();
```

![image](https://user-images.githubusercontent.com/482998/46777970-d24fa680-cd3b-11e8-826a-0167fad9a18a.png)

```bash
$ tsc cli-app/index.ts
node_modules/cac/dist/Cac.d.ts:2:8 - error TS1192: Module '"events"' has no default export.

2 import EventEmitter from 'events';
```

even another manner below will raise same error.

```js
import * as Cac from 'cac';
const cli = Cac.default();

import Cac from 'cac';
const cli = Cac();
```

Though it might be a newbie like question, any suggestions will be appreciated.

**Comment by egoist** [comment]
@mitolog does installing `@types/node` solves the problem?

**Comment by mitolog** [comment]
thanks for such a quick reply.

![image](https://user-images.githubusercontent.com/482998/46780236-c1a42e00-cd45-11e8-8aba-7458729aa3bf.png)

I've added types into my project, but not solved.

**Comment by egoist** [comment]
Fixed in 5.0.13 (https://github.com/cacjs/cac/commit/c087963e8a89c23f6dcde0a430dcd07d4a900666)

**Comment by mitolog** [comment]
@egoist it can transpile now, thanks!
